logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/MessageContent.kt

```diff
@@ -2,8 +2,14 @@ package com.wire.kalium.logic.data.message
 
 sealed class MessageContent {
 
-    data class Text(val value: String): MessageContent()
-    data class Calling(val value: String): MessageContent()
+    data class Text(val value: String) : MessageContent()
 
-    object Unknown: MessageContent()
+    data class Calling(val value: String) : MessageContent()
+
+    sealed class AssetContent : MessageContent() {
+        data class ImageAsset(val value: AssetProtoContent) : AssetContent()
+        data class FileAsset(val value: AssetProtoContent) : AssetContent()
+    }
```

Comment:
What do we gain by this nesting? Can we flatten it to:
```kotlin
data class ImageAsset(val value: AssetProtoContent) : MessageContent()
data class FileAsset(val value: AssetProtoContent) : MessageContent()
```

Reply:
I thought differentiating between `files` and `images` assets will give us an extra degree of redundancy. However since even the protobuff current definition deprecated the `ImageAsset` object in favor of the base `Asset` one, I think we can also treat both as assets on Kalium level.